ui/components/multichain/pages/connections/connections.tsx

```diff
@@ -85,10 +87,9 @@ export const Connections = () => {
     setShowDisconnectedAllAccountsUpdatedToast,
   ] = useState(false);
 
-  const urlParams: { origin: string } = useParams();
-  const securedOrigin = decodeURIComponent(urlParams.origin);
+  const urlParams = useParams<Params>() as Params;
```

Comment:
The cast is unnecessary here as well:
```suggestion
  const urlParams = useParams<Params>();
```